invoice_discount_display_amount/models/account_move_line.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+
+    _inherit = "account.move.line"
+
+    discount_total = fields.Monetary(
+        compute="_compute_discount_amount", string="Discount Total", store=True
+    )
+    price_total_no_discount = fields.Monetary(
+        compute="_compute_discount_amount", string="Total Without Discount", store=True
+    )
+
+    @api.depends("discount", "price_total")
+    def _compute_discount_amount(self):
+        for line in self:
+            price_total_no_discount = 0
+            discount_total = 0
+            if not line.discount:
```

Comment:
can be added as a filtered in the for and make a massive change to those that are not invoice lines and do not have a discount